src/main/java/io/fabric8/maven/docker/util/AnsiLogger.java

```diff
@@ -78,7 +78,13 @@ public void info(String message, Object ... params) {
 
     /** {@inheritDoc} */
     public void verbose(String message, Object ... params) {
-        if (verbose) {
+        if (verbose != null && verbose.contains("build")) {
```

Comment:
I would enable this also when this parameter is set to true, so that the old behaviour with `-Ddocker.verbose=true` still works (and which would switch on just the 'build' verbose part).